README.md

```diff
@@ -109,16 +109,6 @@ LLama 2, Vicuna, Alpaca, Baize, ChatGLM, Dolly, Falcon, FastChat-T5, GPT4ALL, Gu
 
 See a complete list of supported models and instructions to add a new model [here](docs/model_support.md).
 
-#### Use Models from modelscope
-You can use models from www.modelscope.cn, just set environment variable FASTCHAT_USE_MODELSCOPE.
-```
-export FASTCHAT_USE_MODELSCOPE=True
-```
-Example:  
-```
-FASTCHAT_USE_MODELSCOPE=True python3 -m fastchat.serve.cli --model-path qwen/Qwen-7B-Chat --revision v1.1.9
-```
-
```

Comment:
hi  @merrymercy  Why usage of models from modelscope is removed, any other place we can place this doc?

Reply:
@wenmengzhou Could you add the code after here? https://github.com/lm-sys/FastChat/blob/3623d9b980dfa97379db3fa519a11186526219e5/docs/model_support.md?plain=1#L61-L62

We want to keep the main readme short